examples/custom-server/index.mjs

```diff
@@ -0,0 +1,13 @@
+import { unstable_createHandler as createHandler } from '@mui/toolpad';
+import express from 'express';
+
+const app = express();
+
+const appBase = '/my-app';
+const handler = await createHandler({
+  dev: process.env.NODE_ENV === 'development',
+  base: appBase,
+});
+app.use('/my-app', handler.handler);
```

Comment:
So in the example `/my-app` is the path that the whole Toolpad runs under including the editor in dev mode, and `/my-app` is the base path where the production app runs?
If so, should they be different to be more obvious?

Reply:
> including the editor in dev mode

Not the editor, only the application. For editing I'm envisioning the following:

- you start it locally in dev mode.
- you run `yarn toolpad --connect http://localhost:3000/my-app` to open the editor

Reply:
I see, that sounds cool!
Are there cases where the first argument for `app.use` and `appBase` would not be the same, though? Or could they share the same variable?
Anyway that's not important at all so I'm approving already.

Reply:
it's an example. for demonstration purposes this is clearer.